index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="styles/normalize.css" rel="stylesheet">
+  <link href="styles/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-answer-screen">0</div>
+    <div class="calculator-row">
+      <div class="calculator-cell calculator-AC">AC</div>
```

Comment:
each calculator button should be a `<button>` and I would change the `. calculator-cell` classname to `. calculator-button`